server/server.js

```diff
@@ -293,6 +294,36 @@ app.get('/logout', (req, res) => {
     }
 });
 
+app.post('/flic_request', header(['button-serial-number','button-battery-level']).exists(), async (req, res) => {
```

Comment:
can we be more specific with the name of this route? we might be implementing more than one Flic-related route in the future, and 'request' is pretty generic. I'd suggest something like `/flic_button_press` or `/flic_internet_request`